lib/Notes/NotePopupModal/tests/NotePopupModal-test.js

```diff
@@ -49,92 +55,64 @@ describe('NotePopupModal', () => {
     setup();
 
     beforeEach(async () => {
-      sessionStorage.removeItem('@folio/ui-dummy:popUpNoteInfo.providerNoteId');
-      await notePopupModal.whenLoaded();
-    });
-
-    describe('waiting for aXe to run', () => {
-      beforeEach(async () => {
-        a11yResults = await axe.run();
-      });
-
-      it('should not have any a11y issues', () => {
-        expect(a11yResults.violations).to.be.empty;
-      });
+      await sessionStorage.removeItem('@folio/ui-dummy:popUpNoteInfo.providerNoteId');
     });
 
-    it('should display modal label', () => {
-      expect(notePopupModal.modal.label).to.equal('Note for patron');
-    });
+    it('should not have axe(a11y) issues', () => runAxeTest());
```

Comment:
Petty: can we keep using the same phrase, "should not have any a11y issues", to make it easier to grep through the output for common phrases?